taxi/models.py

```diff
@@ -15,7 +16,19 @@ def __str__(self):
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    license_number = models.CharField(
+        max_length=255,
+        unique=True,
+        validators=[
+            RegexValidator(
+                regex=r"^[A-Z]{3}\d{5}$",
+                message="Number of licese must consist "
```

Comment:
There is a typo in the error message: 'Number of licese' should be 'Number of license'.